internal/handlers/tigris/msg_insert.go

```diff
@@ -107,6 +107,12 @@ func insertMany(ctx context.Context, dbPool *tigrisdb.TigrisDB, fp *tigrisdb.Fet
 	var inserted int32
 	var insErrors common.WriteErrors
 
+	// Attempt to insert all the documents in the same request to make insert faster.
+	if err := dbPool.InsertManyDocuments(ctx, fp.DB, fp.Collection, docs); err == nil {
+		return int32(docs.Len()), &insErrors
+	}
+
+	// If the transaction failed, attempt to insert each document separately.
```

Comment:
Does `InsertManyDocuments` fail for any integration test setup? I assume not, but since we have fallback, just wanted to double check :hugs:

Reply:
No, as far as I can see, it doesn't. I expect it to fail for ordered insert tests.

Reply:
Cool, I guess if `InsertManyDocuments` starts to fail for whatever reason, we will be able to see it on tigris docker logs.